data-node/src/main/java/org/graylog/datanode/Configuration.java

```diff
@@ -74,19 +74,19 @@ public class Configuration extends BaseConfiguration {
     private boolean disableNativeSystemStatsCollector = false;
 
     @Parameter(value = "opensearch_location")
-    private String opensearchLocation = "./data-node/bin/opensearch-2.4.1";
+    private String opensearchLocation = "./bin/opensearch-2.4.1";
 
     @Parameter(value = "opensearch_version")
     private String opensearchVersion = "2.4.1";
 
     @Parameter(value = "opensearch_data_location")
-    private String opensearchDataLocation = "./data-node/bin/data";
+    private String opensearchDataLocation = "./bin/data";
 
     @Parameter(value = "opensearch_logs_location")
-    private String opensearchLogsLocation = "./data-node/bin/logs";
+    private String opensearchLogsLocation = "./bin/logs";
 
     @Parameter(value = "opensearch_config_location")
-    private String opensearchConfigLocation = "./data-node/bin/config";
+    private String opensearchConfigLocation = "./bin/config";
```

Comment:
This should be `config` to match the change we did in `misc/datanode.conf` earlier.
```suggestion
    private String opensearchConfigLocation = "config";
```